var/spack/repos/builtin/packages/neurodamus-hippocampus/package.py

```diff
@@ -11,6 +11,6 @@ class NeurodamusHippocampus(NeurodamusModel):
     homepage = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
     git      = "ssh://bbpcode.epfl.ch/sim/models/hippocampus"
 
-    version('develop', git=git, branch='master', submodules=True)
-    version('0.2', git=git, tag='0.2', submodules=True, preferred=True)
-    version('0.1', git=git, tag='0.1', submodules=True)
+    version('develop', git=git, branch='master', submodules=True, clean=False)
```

Comment:
Would be nice to set `clean=False` once in neurodamus-model (overriding version) so that all inherit this setting

Reply:
After conversation we decided it is not easily done due to the definition of Version and the DirectiveMeta which are used in spack. Could be done in the future.